code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/AuthIntegrationManager.java

```diff
@@ -0,0 +1,48 @@
+package io.cattle.platform.iaas.api.auth;
+
+import io.cattle.platform.iaas.api.auth.integration.interfaces.AuthConfig;
+import io.cattle.platform.iaas.api.auth.integration.interfaces.AuthConfigManager;
+import io.github.ibuildthecloud.gdapi.factory.SchemaFactory;
+import io.github.ibuildthecloud.gdapi.model.ListOptions;
+import io.github.ibuildthecloud.gdapi.request.resource.impl.AbstractNoOpResourceManager;
+
+import java.util.ArrayList;
+import java.util.Map;
+import javax.inject.Inject;
+
+public class AuthIntegrationManager extends AbstractNoOpResourceManager {
+
+    Map<String, AuthConfigManager> authConfigManagers;
```

Comment:
It would be map as i would know which config i need so i would be able to get the right manager and call correctly. instead of iterating. Just not Implemented yet. Since i halted dev on this manager.